src/components/organisms/TimingEventModal/TimingEventModal.tsx

```diff
@@ -144,17 +120,6 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
           <div className="form-container">
             <h2>Add experience</h2>
             <form className="form" onSubmit={handleSubmit(onUpdateEvent)}>
-              <div className="TimingEventModal__input-group dropdown-container">
-                <SpacesDropdown
-                  portals={dropdownVenueList}
-                  setValue={setValue}
-                  register={register}
-                  fieldName="room"
-                  parentSpace={parentSpace}
-                  error={errors.room}
-                />
-              </div>
-
```

Comment:
Events can't be moved between spaces right now. I've removed this. There'll be a follow-on PR to tidy up the events stuff